lib/manager/conan/extract.ts

```diff
@@ -49,10 +49,12 @@ export default function extractPackageFile(
             let dep: PackageDependency = {};
             const depName = matches.groups.name;
             const currentValue = matches.groups.version.trim();
+            // conan uses @_/_ as a place holder for no userChannel
```

Comment:
```suggestion
            // conan uses @_/_ as a placeholder for no userChannel
```

Placeholder is one word.

Reply:
Ha! You are a saint